command/daemon.go

```diff
@@ -232,6 +232,10 @@ func daemonCommand(cctx *cli.Context) error {
 		errChan <- ingestSvr.Start()
 	}()
 
+	// Wait for http servers to start.
+	time.Sleep(time.Second)
```

Comment:
surely we don't need a sleep to know that the http server is ready? :)

Reply:
The HTTP servers are started in separate goroutines, and there is not any channel reporting when they are ready.  This was to be reasonably sure that those servers were ready to accept requests before logging the "Indexer ready" message.

Will remove as it is not necessary.